### ISSUE TITLE
Task-1 Neural Network

### ISSUE THREAD

**Issue by shah-suhani** [shah-suhani/WARP-26-INSPECT#1]
Task:
Implement Neural Network on MNIST dataset using pytorch.
Try out different variations based on your learnings 
Figure out how they affect the accuracy .
